cppcache/src/TcrMessage.cpp

```diff
@@ -801,7 +801,7 @@ void TcrMessage::processChunk(const std::vector<uint8_t>& chunk, int32_t len,
       } else if (TcrMessage::CQ_EXCEPTION_TYPE == m_msgType ||
                  TcrMessage::CQDATAERROR_MSG_TYPE == m_msgType ||
                  TcrMessage::GET_ALL_DATA_ERROR == m_msgType) {
-        if (bytes != nullptr) {
+        if (chunk.size()) {
```

Comment:
I think (!chunk.empty()) is more readable. Relying on the decay of `int` to `bool` feels dirty and is less readable.

Reply:
I always try to test positive conditions rather than negative, but am agnostic here - will change